build.sh

```diff
@@ -1,4 +1,16 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if ! (( $# ))
+then
+    echo "usage: $0 <base-href>"
+    exit 1
+fi
+
 npm install
 npm install -g @angular/cli@16.0.0-next.0
 ng build ziti-console-lib
-ng build ziti-console --base-href $1 --deploy-url $1
```

Comment:
pls undo

Reply:
I done gone and undid it. I added a deprecation warning since Angular is threatening to remove `--deploy-url` in a future major release.